test/testcamera.c

```diff
@@ -358,7 +368,19 @@ SDL_AppResult SDL_AppIterate(void *appstate)
 
         /* Update SDL_Texture with last video frame (only once per new frame) */
         if (frame_current && !texture_updated) {
-            SDL_UpdateTexture(texture, NULL, frame_current->pixels, frame_current->pitch);
+            if (frame_current->format == SDL_PIXELFORMAT_MJPG) {
+                /* Note: This is very slow if compiled without optimizations */
```

Comment:
Should we log something when `NDEBUG` is not defined?

Reply:
No, NDEBUG isn't universally defined.